launcher/modplatform/modpacksch/FTBPackInstallTask.cpp

```diff
@@ -320,6 +326,9 @@ void PackInstallTask::downloadPack()
 void PackInstallTask::onModDownloadSucceeded()
 {
     m_net_job.reset();
+    if (m_blocked_mods.length() > 0) {
```

Comment:
```suggestion
    if (!m_blocked_mods.isEmpty()) {
```

Reply:
... I was looking for that method but for some reason my editor told me it didn't exist the first time....